recipes/image-meta-tag/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "0.5.0" %}
+
+package:
+    name: image-meta-tag
+    version: {{ version }}
+
+source:
```

Comment:
Please build from a release rather than the repo. See:

http://conda-forge.github.io/docs/meta.html#build-from-tarballs-not-repos

You can get the tarball url from:

https://github.com/SciTools-incubator/image-meta-tag/releases